Sage/carbon
# chore: roll back version of storybook and its addons

### Proposed behaviour

Revert back versions of `@storybook/react` and all the installed storybook addons to version `6.4.19` due to documentation not loading correctly in latest version.

### Current behaviour

Opening up the documentation of any component in storybook results in: `TypeError: Cannot read properties of undefined (reading 'fonts')`. 

### Checklist




### Additional context

A similar issue was raised with the storybook team, but was closed last year: https://github.com/storybookjs/storybook/issues/12016

### Testing instructions

Pull PR branch to local machine and run storybook - verify component documentation loads correctly and no `TypeError` is thrown.
